src/test/java/com/reviews/demo/impl/ReadingCsvServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package com.reviews.demo.impl;
+
+import com.reviews.demo.dto.CommentDto;
+import com.reviews.demo.service.FileCsvParser;
+import com.reviews.demo.service.impl.CommentFileCsvParser;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ReadingCsvServiceImplTest {
+    private static final String WRONG_FILE_PATH = "random/path";
+    private static final String EMPTY_FILE = "src/test/resources/test_empty_file.csv";
+    private static final String EMPTY_FILE_WITH_HEADER = "src/test/resources/test_empty_file_with_header.csv";
+    private static final String FILE_PATH = "src/test/resources/test_file.csv";
+    private static final String FILE_PATH_WITHOUT_HEADER = "src/test/resources/test_file_without_header.csv";
+    private static final List<CommentDto> EXPECTED_COMMENT_DTOS = List.of(
```

Comment:
```suggestion
    private static final List<CommentDto> expectedCommentDtos = List.of(
```
It's list of mutable objects